.hugo-version

```diff
@@ -1 +1 @@
-0.92.0
+0.123.3
```

Comment:
@clmedders @nick-mon1 we should check in on these versions in POAM.

Reply:
will add in poam doc to capture this!